examples/embeddable_examples/public/react_embeddables/data_table/data_table_queries.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { DataView } from '@kbn/data-views-plugin/public';
+import { buildDataTableRecord } from '@kbn/discover-utils';
+import { DataTableRecord, EsHitRecord } from '@kbn/discover-utils/types';
+import { Filter } from '@kbn/es-query';
+import { i18n } from '@kbn/i18n';
+import { listenForCompatibleApi } from '@kbn/presentation-containers';
+import { apiPublishesDataViews, fetch$ } from '@kbn/presentation-publishing';
+import { BehaviorSubject, combineLatest, lastValueFrom, map, Subscription, switchMap } from 'rxjs';
+import { v4 } from 'uuid';
+import { StartDeps } from '../../plugin';
+import { apiPublishesSelectedFields } from '../field_list/publishes_selected_fields';
+import { DataTableApi, DataTableQueryState } from './types';
+
+export const initializeDataTableQueries = async (
+  services: StartDeps,
+  api: DataTableApi,
+  queryLoading$: BehaviorSubject<boolean | undefined>
+) => {
+  // initialize services
+  const defaultDataViewPromise = services.data.dataViews.getDefault();
+  const searchSourcePromise = services.data.search.searchSource.create();
+  const [defaultDataView, searchSource] = await Promise.all([
+    defaultDataViewPromise,
+    searchSourcePromise,
+  ]);
+  if (!defaultDataView) {
+    throw new Error(
+      i18n.translate('embeddableExamples.dataTable.noDataViewError', {
```

Comment:
remove i18n

Reply:
Also, this message does not align with if statement check for `!defaultDataView`

Reply:
Updated message in e19133b6ad3